tools/revert-release.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# CAUTION!
+# This script does one thing, which is to revert stable tag in WordPress.org svn to the prior tag.
+# It should only be used in extreme emergency cases.
+
+# Check for requirements. Using `jq` to easily parse json. Is there a simpler way to get the latest tags?
+if [ -z $(command -v jq) ]; then
+	echo "This script requires the jq library."
+	read -p "Do you want to install it on your system with Homebrew? [y/N]" -n 1 -r
+	if [[ $REPLY != "y" && $REPLY != "Y" ]]; then
+		exit 1;
+	else
+		brew install jq 2>/dev/null
+		echo "Done!"
```

Comment:
I would exit after this, so the person can start from scratch once they've installed jq.

Reply:
Done in 8fb5f4a0efdbefd0e8d47846a511d2d6383d9653